src/Forms/GroupForm/GroupForm.jsx

```diff
@@ -159,6 +165,15 @@ const GroupForm = ({
           }}
           value={name}
         />
+        <Textarea
+          placeholder={t("screens.allGroups.groupDescription")}
+          onChange={(value) => {
+            setDescription(value);
+          }}
+          value={description}
+          rows={5}
+          maxlength={100}
```

Comment:
isn't 100 characters a bit little?

Reply:
What should the limit be? I can't imagine how long the description text could / should be.

Reply:
I think 250 could be a good value

Reply:
done